docs/pages/blog/tenerife-retreat.md

```diff
@@ -0,0 +1,162 @@
+---
+title: MUI's company retreat in Tenerife: a recap
+description: Our internationally distributed startup gathered on a remote island to get to know each other better. Here's what happened!
+date: 2022-07-25T00:00:00.000Z
+authors: ['samuelsycamore']
+card: true
```

Comment:
I left it `true` too at the aggregation post because of o misunderstanding of what the property means. But `card:true` refers to "banner" image called card.

More context: https://github.com/mui/material-ui/pull/33595#discussion_r929075080

```suggestion
card: false
```